### PR TITLE
site: site code optimization

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | site: site code optimization |
| 🇨🇳 Chinese |          - |

### CODE REVIEW

**Review by afc163 — DISMISSED** [review]

File: .dumi/theme/slots/Header/Logo.tsx [hunk 1]
```diff
@@ -23,17 +23,16 @@ const useStyle = createStyles(({ token, css }) => {
       text-decoration: none;
       display: inline-flex;
       align-items: center;
-
+      column-gap: ${token.marginSM}px;
```
Comment (afc163) [inline .dumi/theme/slots/Header/Logo.tsx:26]: 空行保留。

File: .dumi/theme/slots/Header/Logo.tsx [hunk 2]
```diff
@@ -50,21 +49,16 @@ export interface LogoProps {
...
-      <Link to={utils.getLocalizedPathname('/', isZhCN, search)} className={styles.logo}>
-        <img
-          src="https://gw.alipayobjects.com/zos/rmsportal/KDpgvguMpGfqaHPjicRK.svg"
-          height={32}
-          width={32}
-          alt="logo"
-        />
-        <span className={styles.title}>Ant Design</span>
-      </Link>
-    </h1>
```
Comment (afc163) [inline .dumi/theme/slots/Header/Logo.tsx:67]: h1 为啥删了？

Comment (li-jia-nan) [reply .dumi/theme/slots/Header/Logo.tsx:67]: h1 在这里除了语义之外没有任何用处了，也没有任何样式，删了之后不影响布局

File: .dumi/theme/slots/Header/Logo.tsx (hunk 1 above)
Comment (li-jia-nan) [reply .dumi/theme/slots/Header/Logo.tsx:26]: done

File: .dumi/theme/slots/Header/Logo.tsx (hunk 2 above)
Comment (li-jia-nan) [reply .dumi/theme/slots/Header/Logo.tsx:67]: 那我加回来

File: .dumi/theme/slots/Footer/AdditionalInfo.tsx [hunk 3]
```diff
@@ -19,9 +20,38 @@ const locales = {
...
+    border: 1px solid #ffe58f;
+    background-color: #fffbe6;
+    color: rgba(0, 0, 0, 0.88);
+    padding: 8px 12px;
+    border-radius: 8px;
+    z-index: 9999999999;
+    line-height: 22px;
+    width: 520px;
+    a {
+      color: #1677ff;
```
Comment (afc163) [inline .dumi/theme/slots/Footer/AdditionalInfo.tsx:46]: 用 colorPrimary

Comment (li-jia-nan) [reply .dumi/theme/slots/Footer/AdditionalInfo.tsx]: done

File: .dumi/theme/slots/Header/Logo.tsx (hunk 2 above)
Comment (li-jia-nan) [reply .dumi/theme/slots/Header/Logo.tsx:67]: done

**Review by afc163 — APPROVED** [review]

**Pull request merged**